README.md

```diff
@@ -6,7 +6,7 @@ This repository contains Giant Swarm releases and changelogs.
 
 ### AWS
 
- - [11.1.2](https://github.com/giantswarm/releases/blob/master/release-notes/aws/v11.1.2.md)
+ - [11.1.3](https://github.com/giantswarm/releases/blob/master/release-notes/aws/v11.1.3.md)
```

Comment:
I think we should only add the new release.

Reply:
There is no point linking to the replaced release from the main README imo.

Reply:
There is because we released something and now we release a patch. People use it already and we should reflect the history transparently.

Reply:
I'll add it back and may provide a cleanup PR in the future